tools/pdp-metadata/pdp-metadata.js

```diff
@@ -3,26 +3,21 @@ import fs from 'fs';
 import he from 'he';
 import productSearchQuery from './queries/products.graphql.js';
 import { variantsFragment } from './queries/variants.graphql.js';
-
 const basePath = 'https://www.aemshop.net';
-const configFile = `${basePath}/configs.json?sheet=prod`;
+const configFile = `${basePath}/config.json?sheet=prod`;
```

Comment:
There are no sheets on this file.
```suggestion
const configFile = `${basePath}/config.json`;
```